pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -478,9 +478,9 @@ def get_level_max(self):
 
     def get_width_of_level(self, level: int):
         """
-        Fonction permettant de retourner la largeur d'un niveau sachant root niveau 0
+        Function to return the width of a root level level level 0
```

Comment:
Return the width of a level, considering root level = 0

Reply:
```suggestion
        Return the width of a level, considering root level = 0
```